packages/webapp/__tests__/PostPage.tsx

```diff
@@ -226,6 +226,34 @@ it('should show login on upvote click', async () => {
   expect(showLogin).toBeCalledTimes(1);
 });
 
+it('should check meta tag', async () => {
```

Comment:
I would name these a bit more explicit as:
"should check meta tag with only summary"